source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1521,22 +1512,32 @@ namespace RendererQml
 				uiRow->AddChild(uiElement);
 
 				const auto width = cardElement->GetWidth();
-				if (width != "stretch" && width != "auto" && width != "")
+				
+				if (width.size() > 2 && width.substr(width.size() - 3, 2) == "px")
```

Comment:
do we need to check for this `width.substr(width.size() - 3, 2) == "px"`. I thought the width is always provided as px.. else the json fails?